src/Illuminate/Contracts/View/CanBeCached.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Illuminate\Contracts\View;
+
+interface CanBeCached
```

Comment:
What do you think about renaming this to `Cacheable`?

Reply:
Good idea - I just tried to stick to the `ShouldBeUnique`, `ShouldBeEncrypted`, etc. naming conventions.

Reply:
I know this is just semantics, but "Cacheable" to me feels like it *can* be cached, not that it will.